ingestion-lambda/src/categoryCodes.ts

```diff
@@ -39,6 +39,15 @@ function replacePrefixesFromLookup(
 	const newPrefix = lookup[prefix] ?? prefix;
 	return { prefix: newPrefix, code };
 }
+
+export function processReutersCategoryCodes(original: string[]): string[] {
+	const supportedDestinations: string[] = ['RWS', 'RNA', 'RWSA', 'REULB', 'RBN'];
+
+	return original
+		.filter((_) => !supportedDestinations.includes(_))
```

Comment:
is this right? this looks like it'll drop the supported destinations; should it be the opposite?

Reply:
Correct :)